frontend/uddhar-frontend/src/coordinator/pages/CoordinatorDashboard.jsx

```
@@ -62,15 +56,8 @@ const CoordinatorDashboard = ({ activeSection }) => {
   const onClickEventHandler = async (info) => {
     // set current disaster
     setCurrentEvent(info);
+    toast.info('Seeing details of: ' + info.title, { duration: 2000, color: 'green' });
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)